specfile/changelog.py

```diff
@@ -223,6 +247,22 @@ def delete(index):
         else:
             delete(i)
 
+    def get_entries_since(self, evr: str) -> List[ChangelogEntry]:
```

Comment:
What about implementing both limits (`from`/`to`, `before`/`after`, `since`/`to`...) and having `to`/`before` as optional with the current EVR as a default.

Reply:
It took me a while to get what you mean :slightly_smiling_face: To be sure, are you talking about having two arguments (e.g. since, until) with defaults instead of one (evr)?

Reply:
I think that would make sense, we could make it work a bit like list slicing. E.g. you would be able to specify: give me all changelog entries between 1.2.3 and 2.3.1. Both of these arguments could be optional meaning that one could use `since=None, until='1.2.3'` which would mean all releases until `1.2.3`.

Reply:
I agree that it would be nice, I'm just not 100% sure this is what Franta had in mind.

Reply:
Yes, exactly. That was what I had in mind.

>  Both of these arguments could be optional meaning that one could use since=None, until='1.2.3' which would mean all releases until 1.2.3.

+1, we can use beginning as a default for `since` and the current version for `until`.